### ISSUE TITLE
Remove references to discussing changes on Discord

### ISSUE THREAD

**Issue by Asartea** [TheOdinProject/theodinproject#3892]
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `Bug: brief description of bug` format, e.g. `Bug: Lesson complete button does not update on click`

The following checkbox is OPTIONAL:

-   [x] I would like to be assigned this issue to work on it

<hr>

**1. Description:**
`CONTRIBUTING.md` still mentions in the section on minor changes:
> You can just open an issue or a PR in the appropriate repo **rather than needing to bring it up on our Discord server** (emphasis mine)
My understanding is that TOP is moving/has moved to fully using Github `issues` (and other on GitHub discussion features) instead of the on Discord `#suggestions-bugs` channel, which also has been retired. `CONTRIBUTING.md` should probably be updated to remove this reference.

This change needs to be made both in this repo and in https://github.com/TheOdinProject/.github

**Comment by ManonLef** [comment]
Hi @Asartea and thanks for taking this on. As discussed on Discord, there are some more references to this retired Discord channel. 

Don't worry about the onboarding docs nor the wikis (since the wikis can't be edited like the repo), but the FAQ.MD in top meta can actually be changed. Changing that one does require the discord FAQ channel to be updated as a task after merge so I will experiment with a tasklist after this reply.  

Would you mind changing all of them to mention and point to the corresponding repo issues link?

**Comment by Asartea** [comment]
> Would you mind changing all of them to mention and point to the corresponding repo issues link?

The one issue I could see with this is that the `CONTRIBUTING.md` in this repo also gets referenced in several other TOP repos, and the one in `.github` is auto linked in the issues and pr tab for every TOP repo that doesn't have its own.

Given that, I'm not sure if it makes sense to say update the one in this repo to point to https://github.com/TheOdinProject/theodinproject/issues, when a large amount of the time people are reading it they really want to create a issue in the curriculum repo. I think it would be clearer to just strip "rather than needing to bring it up on our Discord server" from that line

Also, the only place `suggestions-bugs` gets mentioned in the FAQ is with a reference to a dark theme in beta, but I'm not sure if thats still accurate? Git blame points to you adding that 7 months ago (https://github.com/TheOdinProject/top-meta/commit/ca412389898e8c49620a379ce175d07a407fdc23), but the commit description suggests to me that was copied from the `#faq` channel on Discord; is this still relevant?

**Comment by ManonLef** [comment]
> Also, the only place suggestions-bugs gets mentioned in the FAQ is with a reference to a dark theme in beta, but I'm not sure if thats still accurate? Git blame points to you adding that 7 months ago (https://github.com/TheOdinProject/top-meta/commit/ca412389898e8c49620a379ce175d07a407fdc23), but the commit description suggests to me that was copied from the #faq channel on Discord; is this still relevant?

The FAQ didn't get merged until quite recently. I've asked the same question before merge about the dark mode beta back then but I can't exactly remember if it got answered. However, I just heard it's definitely not in beta anymore so we can safely remove that reference and remove everything after the `keep in mind` part. 

> I think it would be clearer to just strip "rather than needing to bring it up on our Discord server" from that line

I agree, feel free to reword it to something more appropriate. But just to be sure: When I mentioned *"Would you mind changing all of them to mention and point to the corresponding repo issues link?"* I meant the [discord reference](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md#further-help) at the bottom of the article. Which I think could do with the change stated above not necessarily linking to the `issues`, but mentioning openingor filing an issue instead. 


Another option would be to add a link to [GH's guide on how to create an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue)

**Comment by KevinMulhern** [comment]
This all looks great to me @ManonLef, @Asartea. The only thing that comes to mind, is we could maybe provide links to the issue for our most popular repos. But, no worries if you don't want to go down that route. Whatever you decide is good with me. If an issue happens to be made in the wrong repo, it's easy to transfer, it won't be a big deal if it happens.

**Comment by Asartea** [comment]
@KevinMulhern would you also perhaps happen to know if TOP still considers dark mode to be in beta?

**Comment by ChargrilledChook** [comment]
@Asartea No, that line is no longer relevant. I believe that was referring to the implementation added after the Chrome add-on but  before we switched to Tailwind.

**Comment by Asartea** [comment]
Think this should be ready for work?

**Comment by Asartea** [comment]
> What do you think about a small list linking to the issues of the most popular repos?


Yeah, that seems like a good idea, although I'm not sure where it would best fit

> Another option would be to add a link to [GH's guide on how to create an issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) so it becomes something like "Please let us know if you require any further help with any of the steps in this guide by creating an issue in the relevant repository. If you need help on creating an issue `link to guide`".

Circling back to this: I think it would be best to also still include a generic link to the Discord (probably just using the main invitation link), as Discord as a platform for discussing stuff like "hey, I want to do this, how do I make a PR" probably still has merit

**Comment by ManonLef** [comment]
I see what you mean @Asartea 
At the same time, I think the guide is very elaborate and if someone has trouble with parts of it, that is valuable feedback. It might easily be missed by maintainers in Discord as opposed to if it's addressed as an issue. 

How would you feel about updating the [Creating an issue](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md#creating-an-issue) section and possibly include the links to our repos there? The GH guide is already in that section I notice now.

Then at the bottom in the `Further Help` section we could change it to for example *"Please let us know if you require any further help with any of the steps in this guide by opening an issue (link to the creating an issue header)"* with the addition of something along the lines of *"for more general contributing questions not covered in this guide you can also join our discord"*
Just some ideas.

Feel free to take some time and pick out the things you think fit best. We can always tweak it when it comes to reviewing.

**Comment by ManonLef** [comment]
FYI; I also updated the FAQ on discord

**Comment by Asartea** [comment]
Should be all done
